### PR TITLE
Ft/common req auth

### PR DESCRIPTION
**Pull request by jmunoznaranjo**
Related:
Vault: https://github.com/scality/Vault/pull/359 
Arsenal: https://github.com/scality/Arsenal/pull/82

### CODE REVIEW

File: bin/vaultclient [hunk 1]
```diff
@@ -10,8 +15,38 @@ program.option('--host [HOST]', 'host where Vault is running (default to ' +
...
+    if (!configObject.accessKey) {
+        throw new Error('invalid access key');
+    }
+    if (!configObject.secretKeyValue) {
+        throw new Error('invalid secret key value');
+    }
+}
 
+function readConfigFile(configFilePath) {
+    let actualPath = path.join(homedir(), '.vaultclient.conf');
```
Comment (alexandre-merle) [inline bin/vaultclient]: i think it need to be '/vaultclient.conf', right ? :)

**Comment by alexandre-merle** [comment]
LGTM :+1:

**Comment by jmunoznaranjo** [comment]
history squash, please review

**Comment by AntoninCoulibaly** [comment]
:+1:

**Pull request merged**